docs/getting-started/getting-started-with-codacy.md

```diff
@@ -25,23 +23,29 @@ Now, you'll need to join one or more organizations that contain your repositorie
 
 To start adding your repositories, select one of the organizations and click **Go to add repositories**.
 
-## 4. Add repositories {: id="add-repositories"}
+## 3. Add repositories {: id="add-repositories"}
 
 As a final step, add the repositories that you wish to analyze. Codacy will start the first analysis and set up everything required to ensure your next commits on those repositories are analyzed.
 
 ![Adding repositories](images/getting-started-add-repository.png)
 
-## You're all set! 🎉 {: id="all-set"}
-
 Codacy begins an initial analysis as soon as you add a repository, and displays an overview of the code quality of your repository when the analysis is complete.
 
-After that, you can continue to explore and configure Codacy for your repository:
+![Repository dashboard](../repositories/images/repository-dashboard.png)
+
+## 4. Tweak your repositories
+
+ [Check the static analysis results](../repositories/issues-view.md) on the Issues page to explore the initial analysis results, and tweak the Codacy configurations for your repositories:
```

Comment:
(again, scope but here it goes anyway, Paulo 😂) Maybe would be cool to mention that this initial analysis is based on Codacy defaults and encourage users to tweak the patterns.